chrome/manifest.json

```diff
@@ -22,7 +22,9 @@
     "cookies",
     "declarativeNetRequestWithHostAccess",
     "webRequest",
-    "storage"
+    "storage",
+	"contextMenus",
+	"tabs"
```

Comment:
@IcedRooibos is this permission necessary? We had it and reverted due to #13

Reply:
Good point, not something I had considered - I shall test this along with the other change over the weekend.